src/ngx-auto-scroll.directive.ts

```diff
@@ -8,7 +8,7 @@ export class NgxAutoScroll implements AfterContentInit, OnDestroy {
     @Input("observe-attributes") public observeAttributes: string = "false";
 
     private nativeElement: HTMLElement;
-    private isLocked: boolean = false;
+    public isLocked: boolean = false;
```

Comment:
Why you made this public?

Reply:
There was a case where I actually needed this and I think other users might need it too. Let's assume that you have a button to take you to the bottom of the page. And you want to hide it if you're already there. This property can help you. But now that I'm thinking I guess it will be better to expose it through a getter. I'll do the change.